ui/src/app/views/page-template-signed-out/component.ts

```diff
@@ -1,13 +1,32 @@
-import {Component, Input} from '@angular/core';
+import {Component, Input, OnChanges, OnInit} from '@angular/core';
 
 @Component({
   selector: 'app-page-template-signed-out',
   templateUrl: './component.html',
   styleUrls: ['./component.css']
 })
-export class PageTemplateSignedOutComponent {
+export class PageTemplateSignedOutComponent implements OnChanges, OnInit {
   @Input() imageSrc = '';
+  @Input() smallerImageSrc = '';
   headerImg = '/assets/images/logo-registration-non-signed-in.svg';
+  templateElement: any;
 
   constructor() {}
+
+  ngOnInit() {
+    this.templateElement = document.getElementsByClassName('template')[0];
```

Comment:
Why go with the native dom api over `ViewChild` and `Renderer2`?

Reply:
No particular reason. I can totally change that, but it does need the class anyway, so this avoids adding a tag as well.